pkg/util/uploaderconfig/uploaderconfig.go

```diff
@@ -0,0 +1,54 @@
+package uploaderconfig
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+
+	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+)
+
+const (
+	parallelFilesUpload = "ParallelFilesUpload"
+	writeSparseFiles    = "WriteSparseFiles"
+)
+
+func StoreBackupConfig(config *velerov1api.UploaderConfigForBackup) map[string]string {
+	data := make(map[string]string)
+	data[parallelFilesUpload] = strconv.Itoa(config.ParallelFilesUpload)
+	return data
+}
+
+func StoreRestoreConfig(config *velerov1api.UploaderConfigForRestore) map[string]string {
+	data := make(map[string]string)
+	if config.WriteSparseFiles != nil {
+		data[writeSparseFiles] = strconv.FormatBool(*config.WriteSparseFiles)
+	}
+	return data
+}
+
+func GetBackupConfig(data map[string]string) (velerov1api.UploaderConfigForBackup, error) {
```

Comment:
I'm not sure if this is a good approach, there may be settings in PVB that is meaningful to the uploader but is not exposed in backup CR.

Reply:
updated